### ISSUE TITLE
[Exact name of vehicle, as displayed in game]: Bugatti Chiron

### ISSUE THREAD

**Issue by Blurrr5** [fkk-cz/noire_vehicles#1613]
### Is there an existing issue for this?

- [X] I have searched the existing issues

### Current Behavior

The Bugatti Chiron is 2.4 million and during the test drive I took it up the freeway and the top speed is only around 150 mph on flat. I understand the QOL adjustments that eventually every car is going to be affected by but there are many cars that are way lower in price that go the same if not a little bit faster in test drives than the Bugatti Chiron.  

### Expected Behavior

I would expect for a car that costs 2.4 million to go at least 180mph on flat because I really do want to purchase this car as I do love the looks of it but just cannot spend 2.4 million on a car that only goes 150mph on flat.

### Steps To Reproduce

Increase the top speed to a minimum of 180mph on flat

### Anything else?

No

**Comment by Ell34** [comment]
Have to remember in test drive it doesn't have upgrades, so once it has them it probably will hit 160-170 flat

**Comment by Blurrr5** [comment]
Still 160-170 for a 2.4 million dollar car is still not good

**Comment by LPLips7** [comment]
- Scuba Has confirmed the new Chiron and the new Sian Roadster will be tweaked. The Bugatti Chiron will probably go 190 or somewhere around there. The handling, speed and acceleration needs tweaking

**Comment by Ell34** [comment]
> * Scuba Has confirmed the new Chiron and the new Sian Roadster will be tweaked. The Bugatti Chiron will probably go 190 or somewhere around there. The handling, speed and acceleration needs tweaking

Awesome

**Comment by Blurrr5** [comment]
Is there an update on the Bugatti Chiron being tweaked?

**Comment by Competent1** [comment]
This vehicle has been in the simulator/shop for many hours getting tweaked on constantly. I drove this vehicle for a few hours and after a lot of adjustments, the issues you have stated in this GitHub Vehicle Issue should _no longer_ be an issue with this vehicle once it is merged over. I appreciate and thank you for making this issue on this vehicle as it was not performing up to the standards it should've been. If there are any further issues regarding this vehicle, or any other vehicle, do make another issue and it can get looked at in the simulator/shop.
